lib/ask/runtime/broker.ex

```diff
@@ -303,15 +316,20 @@ defmodule Ask.Runtime.Broker do
   end
 
   defp sync_step_internal(session, reply, session_mode) do
-    respondent = session.respondent
-
     transaction_result = Repo.transaction(fn ->
       try do
+        session = if reply == Flow.Message.answer() do
+          update_respondent_disposition(session, "contacted")
+        else
+          update_respondent_disposition(session, "started")
```

Comment:
No, it's not being created.

That condition will only be met if the delivery notification of an sms is missed. And only for that mode.

I think that having this missing transition is clearly an indication of something being wrong somewhere. I wouldn't fake it. I'd rather use it to try to find the problem.